### DISCUSSION TITLE
ctf_skill/makeChallenge/

### DISCUSSION THREAD

**Discussion by giscus** [ProbiusOfficial/Hello-CTF#25] [Show and tell]
# ctf_skill/makeChallenge/

开源免费、新手友好的CTF(Capture The Flag,夺旗赛)入门教程

https://ctf.tj.cn/ctf_skill/makeChallenge/

**Comment by NTLoner** [comment]
受益匪浅，感谢tj和巨魔师傅

**Reply by Randark-JMT** [reply]
多感谢tjjj就好，我给他端茶倒水的😁

**Reply by ProbiusOfficial** [reply]
又开始了是吧

**Comment by Abyssun** [comment]
受益匪浅，感谢tj和巨魔师傅

**Reply by Randark-JMT** [reply]
多感谢tjjj就好，我给他端茶倒水的😁😁😁

**Comment by Abyssun** [comment]
提问，我的dockerhub登陆不上，也搜索不到其他师傅的镜像是什么原因

**Reply by Randark-JMT** [reply]
Dockerhub近期受到GFW影响，据说已经开始被影响，请考虑上代理

**Reply by Abyssun** [reply]
这边已经上代理了，感觉好奇怪，dockerhub处于一种完全用不了的状态，但是我在Ubuntu中docker login倒是正常的

**Reply by Randark-JMT** [reply]
> 这边已经上代理了，感觉好奇怪，dockerhub处于一种完全用不了的状态，但是我在Ubuntu中docker login倒是正常的
> 

具体是指什么情况呢？是否将dockerd引擎的配置也指定了代理参数？
